src/python/pants/python/python_setup.py

```diff
@@ -216,6 +216,16 @@ def register_options(cls, register):
             help="Tailor pex_binary() targets for Python entry point files.",
         )
 
+        register(
+            "--macos-big-sur-compatibility",
+            type=bool,
+            default=False,
```

Comment:
Should we configure this as True in our pants.toml or in some more specific way such that releases do the right thing?

Reply:
Oh, oops, yes we should. It seems like this should usually be True, but I'm not confident enough in that fact for that to be the default.